app/qml/inputs/MMTextBubblesEditor.qml

```diff
@@ -0,0 +1,168 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Controls.Basic
+import "../components"
+
+MMAbstractEditor {
+  id: root
+
+  property var parentValue: parent.value ?? ""
+  property bool parentValueIsNull: parent.valueIsNull ?? false
+  property bool isReadOnly: parent.readOnly ?? false
+
+  readonly property real itemHeight: 36 * __dp
+  readonly property int rows: 3
+  required property ListModel featuresModel
```

Comment:
Please encapsulate private properties used only in one file in a separate `QtObject` at the end of the file to make the file more readable https://github.com/MerginMaps/mobile/blob/b3e30023de8a75a09deaa246ead1b5ec2f2a89db/docs/code_convention.md?plain=1#L151